deployer/src/deployment/run.rs

```diff
@@ -423,13 +416,12 @@ async fn run(
         .unwrap()
         .into_inner();
 
-    info!("starting service");
     let response = runtime_client.start(start_request).await;
 
     match response {
         Ok(response) => {
             if response.into_inner().success {
-                info!(DEPLOYER_RUNTIME_START_RESPONSE);
+                info!("{}", DEPLOYER_RUNTIME_START_RESPONSE);
```

Comment:
Just curious, what is the benefit of this change?

Reply:
Logging `string` vs logging `{VARIABLE_NAME="string"}`